lotto.js

```diff
@@ -0,0 +1,98 @@
+let winningNumbers = [];
+let buyCount = 0;
+
+function LottoNumber() {
+  const numbers = new Set();
+  while (numbers.size < 6) {
+    const rand = Math.floor(Math.random() * 45) + 1;
+    numbers.add(rand);
+  }
+  return Array.from(numbers).sort((a, b) => a - b);
+}
+
+function WinningNumbers() {
+  const result_list = document.querySelector(".result");
+  const resultNumbers = LottoNumber();
+  const old_result_list = document.querySelector(".result .balls");
+  if (old_result_list) old_result_list.remove();
+  result_list.innerHTML += `
+    <div class="balls">
+      <div class="ball" id="yellow">${resultNumbers[0]}</div>
+      <div class="ball" id="blue">${resultNumbers[1]}</div>
+      <div class="ball" id="red">${resultNumbers[2]}</div>
+      <div class="ball" id="gray">${resultNumbers[3]}</div>
+      <div class="ball" id="green">${resultNumbers[4]}</div>
+      <div class="ball" id="yellow">${resultNumbers[5]}</div>
+    </div>
+  `;
+}
+
+document.getElementById("buy_count").addEventListener("input", () => {
+  const count = document.getElementById("buy_count").value;

```

Comment:
```suggestion
document.getElementById("buy_count").addEventListener("input", (e) => {
  const count = e.target.value;
```

이런식으로 사용 가능해요! 여기서 e가 의미하게 무엇인지 학습해보세요!

Reply:
e는 이벤트 객체로 이벤트가 발생했을 때 브라우저가 자동으로 만들어주는 객체입니다.
이벤트가 발생한 HTML 요소를 가르키는 데 위와 같은 상황에선 e.target.value = input창에 입력된 값입니다.
e에 대해 학습완료했습니닷!